django_loci/tests/base/test_selenium.py

```diff
@@ -11,10 +11,13 @@
 class BaseTestDeviceAdminSelenium(
     SeleniumTestMixin, TestAdminInlineMixin, TestLociMixin
 ):
+    def _create_device(self):
+        self.find_element(by=By.NAME, value='name').send_keys('11:22:33:44:55:66')
+
     def test_create_new_device(self):
         self.login()
         self.open(reverse(self.add_url))
-        self.find_element(by=By.NAME, value='name').send_keys('11:22:33:44:55:66')
+        self._create_device()
```

Comment:
Is there a specific reason for creating this new one-line function? I also noticed that it’s not being used in multiple places. Also as it is just filling the form shoul we need to name this function `_create_device`?

Reply:
for device admin in `openwisp-controller` there is another required field - mac address. So i just kind of kept the logic for device specific fields in another function to avoid rewriting the complete for `openwisp-controller`.

Reply:
Renamed the function to `_fill_device_form`

Reply:
@DragnEmperor there's no point in creating methods if the logic is not shared across different parts of the code or unless the logic is too big and needs to be split to increase readability.